creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -85,7 +93,24 @@ const Metrics = Object.freeze({
         5
       )
     }
-  }
+  },
+  // Add histogram for each job in the state machine queues
+  ...Object.fromEntries(
+    Object.values(JOB_NAMES).map((jobName) => [
+      MetricNames[`STATE_MACHINE_${jobName}_JOB_DURATION_SECONDS_HISTOGRAM`],
+      {
+        metricType: MetricTypes.HISTOGRAM,
+        metricConfig: {
+          name: MetricNames[
+            `STATE_MACHINE_${jobName}_JOB_DURATION_SECONDS_HISTOGRAM`
+          ],
+          help: `Duration in seconds for a ${jobName} job to complete`,
+          labelNames: ['uncaughtError'], // Whether the job completed (including with a caught error) or quit unexpectedly
+          buckets: [1, 5, 10, 30, 60, 120] // 1 second to 2 minutes
```

Comment:
we'll likely have to tweak these buckets after getting some data in here, but thats fine - seems like the expected prometheus pattern anyway?